### ISSUE TITLE
Brand Refresh: Iconography

### ISSUE THREAD

**Issue by mcturco** [asyncapi/brand#10]
This issue will be specifically geared towards defining a set of icons for use across AsyncAPI.

**Comment by mcturco** [comment]
To kick off this issue, just wanted to leave some thoughts that I had about how to organize iconography sets. I am envisioning that there will be two "buckets" of icons: 

1. Marketing
2. Tooling

This also directly resonates with my recent thought of creating two separate design systems: 1 for Marketing (website, docs), and 1 for Tooling (products, UI). This is because the application of styles will be a little bit different depending on medium. For example, on the website we might have icons to represent things like `Documentation` or `Use Cases` but on Tooling products, we most likely will not have a use for icons like that. 

Within those two buckets, we can definitely have sub-categories to make it easier to choose from. Before we can define the specific icons that we will need however, I think it would be great to perform an audit of the icons that we are currently using in those two buckets. Then, we can kinda get an idea of what icons we can streamline and improve upon and then make our own kit.

Would love some help on this issue if there are any visual/product designers out there looking to make a contribution! 💪

**Comment by Mayaleeeee** [comment]
Hi all 👋

Closing this issue as it's now outdated and doesn't reflect our current design system and brand direction.

We've opened a fresh and more scoped issue that replaces this one: [Create a Unified Iconography System for Marketing & Tooling](https://github.com/asyncapi/brand/issues/119)

That new task covers auditing, categorizing, and designing icons for both our marketing efforts and our tooling products, with clearer deliverables and alignment to the current AsyncAPI Design System.
